src/components/Songs.jsx

```diff
@@ -41,61 +43,93 @@ export default class Songs extends Component {
+                return;
+            default:
+                return;
+        }
+    }
+
     render() {
         const { x, y } = this.state;
         const style = {
             borderRadius: 0,
             border: "2px solid black",
             padding: "2em"
         };
+        let sizeClass = "size-" + this.state.size;
         // Used for testing
         // let tempSongs = this.state.songs.slice(0, 80);
         return (
-            <Container
-                style={{
-                    padding: "2px",
-                    display: "flex",
-                    flexWrap: "wrap",
-                    justifyContent: "center"
-                }}
-            >
+            <Container>
                 <Navigation />
-                {/* <Embed
-                    id="7Af6b9-yqa8"
-                    placeholder={`https://img.youtube.com/vi/7Af6b9-yqa8/mqdefault.jpg`}
-                    source="youtube"
-                /> */}
-                <div className="song-header-container" />
-
-                {/* <Image.Group size='small' style={{marginTop: '5px', backgroundColor: 'black'}}> */}
-                {this.state.songs.map(song => {
-                    return (
-                        //<img src={"2009/" + song.title.replace(/\s/g,'_') + "_small.png"}
-                        <AlbumCanvas
-                            width={300}
-                            backgroundImage={"/" + song.imagePathSmall}
-                            song={song}
-                            songnumber={song.number}
-                            list={true}
-                        />
-                    );
-                    //   <Popup size='tiny' key={song._id} style={style} hoverable inverted trigger={<AlbumCanvas width={100} height={100} images={[]} song={song} songnumber={song.number} />}>
-                    //     <Popup.Header>{song.title}</Popup.Header>
-                    //     <Popup.Content>
-                    //       <Embed style={{width:'340px', height: '160px'}}
-                    //         id={song.videoid}
-                    //         placeholder={`https://img.youtube.com/vi/${song.videoid}/mqdefault.jpg`}
-                    //         source='youtube'
-                    //       />
-                    //       {this.renderKey(song)}
-                    //     </Popup.Content>
-                    //   </Popup>
-                    // )
-                    // })}
-                    {
-                        /* </Image.Group> */
-                    }
-                })}
+                <div className="filter-navigation">
+                    <div className="filter-navigation-inner">
+                        <ul className="filter-size">
+                            <li>Size</li>
+                            {this.renderSizes()}
+                        </ul>
+                    </div>
+                </div>
+                <div className={"song-list-container " + sizeClass}>
```

Comment:
Interpolation might be a better choice here and a few other places but I also see how that gets confusing in jsx based classes. 
{"song-list-container " + sizeClass} vs {`song-list-container ${sizeClass}`} is kinda a tossup on readability.

Reply:
yeah, prob.

Reply:
Ah shit, maybe this needs lots more work actually.

Reply:
Nah, it's not that big of a deal. As I said it's a tossup but we should figure out a style and code guide at some point to decide these issues.